docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
just curious is there a reason to redefine the `VolumeCreateRequest` type? We could just use `types.VolumeCreateRequest` everywhere, right? 

I would prefer the latter as that's what we do elsewhere in the project. 

If we decide to do the current way, it might be better to keep this type in api package itself i.e. here https://github.com/contiv/volplugin/pull/320/files#diff-604f4192deea5adf9c159e5f88f64630L21 , that will limit the change just to one file